salt/modules/lxc.py

```diff
@@ -989,6 +1045,10 @@ def init(name,
     if users is None:
         users = []
     dusers = ['root']
+    # think to touch the default user generated by default ubuntu template which
+    # has a really unsecure password too ...
+    if __grains__['os'] in ['Ubuntu'] and 'ubuntu' not in users:
+        dusers.append('ubuntu')
```

Comment:
Good catch, I added a step to my pull request which runs ``id ubuntu``, as doing that is quicker than using getent and parsing out all the extraneous data.